examples/wordcount/src/main/java/com/hazelcast/jet/examples/wordcount/WordCount.java

```diff
@@ -72,18 +72,19 @@ public static void main(String[] args) throws Exception {
     /**
      * This code illustrates a few more things about Jet, new in 0.5. See comments.
      */
-    private void go() {
+    private void go() throws InterruptedException {
         try {
             setup();
-            System.out.print("\nCounting words... ");
+            System.out.println("\nCounting words... ");
             long start = System.nanoTime();
             Pipeline p = buildPipeline();
             Observable<Entry<String, Long>> observable = jet.getObservable(COUNTS);
             observable.toFuture(s -> s.collect(toMap(Entry::getKey, Entry::getValue)))
                     .thenApply(WordCount::checkResults)
                     .thenAccept(WordCount::printResults);
             jet.newJob(p).join();
-            System.out.print("done in " + TimeUnit.NANOSECONDS.toMillis(System.nanoTime() - start) + " milliseconds.");
+            System.out.println("done in " + TimeUnit.NANOSECONDS.toMillis(System.nanoTime() - start) + " milliseconds.");
+            Thread.sleep(5_000);
```

Comment:
Is there any `await()` available? Forced `sleeps` don't provide best experience.